ansible/files/envoy_config/remove_apikey_query_parameter.lua

```diff
@@ -0,0 +1,5 @@
+function envoy_on_request(request_handle)
```

Comment:
Add comment describing why we're using this function. If possible, a test capturing the intended behaviour.

Reply:
Added comment and tests.